[JEP-226] Improve migration section (#292)
jenkinsci/jep: jep/226/README.adoc

When a plugin is configured and enabled, the fingerprints are transferred from the local storage to the external storage.

This is followed by clearing the local database of fingerprints from Jenkins HOME ONLY if the writes are successful.

> Following new methods are introduced in the `FingerprintStorage` class:

Reviewer: ```suggestion
We have the following new methods in the `FingerprintStorage` class:
```

Author: Proposed new design, so resolving this
